README.md

```diff
@@ -0,0 +1,30 @@
+# rke-tools
+
+## About
+
+The container image `rancher/rke-tools` is used in RKE (`rancher/rke`) as:
```

Comment:
Is not used but it includes the binaries and tools for what you explain later, or?